testcafe/tests/drive/classification_scenario.js

```diff
@@ -0,0 +1,395 @@
+import {
+  FILE_FROM_ZIP_PATH,
+  FILE_PDF,
+  FILE_TXT,
+  maskDriveFolderWithDate,
+  maskDeleteFolder,
+  maskMoveModale
+} from '../helpers/data'
+
+import PrivateDrivePage from '../pages/drive/drive-model-private'
+const privateDrivePage = new PrivateDrivePage()
+import Modal from '../pages/drive/drive-modal-model'
+const moveModale = new Modal()
+
+//Scenario const
+const FEATURE_PREFIX = 'ClassicationScenario'
+
+const FIXTURE_INIT = `${FEATURE_PREFIX} 1- Prepare Data`
+const TEST_CREATE_FOLDER = `1-1 Create Folders`
+const TEST_UPLOAD = `1-2 Upload file`
+const TEST_UPLOAD_DD = `1-3 Drag&Drop file (upload)`
+
+const FIXTURE_CLASSIFICATION = `${FEATURE_PREFIX} 2- Classification actions on file`
+const TEST_RENAME_FILE = `2-1 Rename file`
+const TEST_MOVE_FILE_CANCEL = `2-2 Move Modal and Cancel`
+const TEST_MOVE_FILE = `2-3 Move file`
+const TEST_MOVE_FOLDER = `2-4 Move folder`
+
+const FIXTURE_TRASH = `${FEATURE_PREFIX} 3- Trash and Restore`
+const TEST_DELETE_FOLDER = `3-1 Delete Folder`
+const TEST_RESTORE_FOLDER = `3-2 Restore Folder`
+const TEST_DELETE_FOLDER_FROM_DRIVE = `3-3 Delete Folder from Drive`
+const TEST_NAVIGATE_IN_TRASH = `3-4 Navigate in Trash`
+const TEST_EMPTY_TRASH = `3-5 Empty Trash`
+
+// const FIXTURE_CLEANUP = `${FEATURE_PREFIX} 3- Cleanup Data`
+// const TEST_DELETE_FOLDER = `3-1 Delete Folder`
+
+//************************
+//Tests when authentified
+//************************
+fixture`${FIXTURE_INIT}`.page`${TESTCAFE_DRIVE_URL}/`
+  .before(async ctx => {
+    await initVR(ctx, SLUG, FIXTURE_INIT)
+  })
+  .beforeEach(async t => {
+    console.group(`\n↳ ℹ️  Loggin & Initialization`)
+    await t.useRole(driveUser)
+    await privateDrivePage.waitForLoading()
+    console.groupEnd()
+  })
+  .after(async ctx => {
+    await ctx.vr.checkRunStatus()
+  })
+
+test(`${TEST_CREATE_FOLDER}`, async t => {
+  await t.maximizeWindow() //Real fullscren for VR
+  console.group(`↳ ℹ️  ${FEATURE_PREFIX} : ${TEST_CREATE_FOLDER}`)
+
+  await privateDrivePage.addNewFolder(
+    `${FEATURE_PREFIX}-Folder1`,
+    `${FEATURE_PREFIX}/${TEST_CREATE_FOLDER}-Folder1-1`
+  )
+  await t.fixtureCtx.vr.takeScreenshotAndUpload(
+    `${FEATURE_PREFIX}/${TEST_CREATE_FOLDER}-Folder1-2`
+  )
+
+  //Create 2nd Folder
+  await privateDrivePage.addNewFolder(
+    `${FEATURE_PREFIX}-Folder2`,
+    `${FEATURE_PREFIX}/${TEST_CREATE_FOLDER}-Folder2-1`
+  )
+  await t.fixtureCtx.vr.takeScreenshotAndUpload(
+    `${FEATURE_PREFIX}/${TEST_CREATE_FOLDER}-Folder2-2`
+  )
+  console.groupEnd()
+})
+
+test(`${TEST_UPLOAD}`, async t => {
+  console.group(
+    `↳ ℹ️  ${FEATURE_PREFIX} : ${TEST_UPLOAD} (in "${FEATURE_PREFIX}-Folder2" folder)`
+  )
+  await privateDrivePage.goToFolder(`${FEATURE_PREFIX}-Folder2`)
+  await t.fixtureCtx.vr.takeScreenshotAndUpload(
+    `${FEATURE_PREFIX}/${TEST_UPLOAD}-1`
+  )
+  await privateDrivePage.uploadFiles([`${FILE_FROM_ZIP_PATH}/${FILE_PDF}`])
+  await t.wait(5000)
```

Comment:
This is to wait for the thumbnails?

Reply:
yes, but only before merging https://github.com/cozy/cozy-drive/pull/1556

Reply:
Ok! It doesnt have to be in this PR, but it would be good to use a variable like `THUMBNAIL_DELAY` instead of a hard-coded value like that. Not only is it easier to change, it also explains why the timeout is there :)